### ISSUE TITLE
Add link on website to download old version

### ISSUE THREAD

**Issue by massivelines** [desktop/desktop#3045]
Love the new direction but too many of the features that I use everyday are missing or hard to get to.

I used the old version as a hub to work on my projects, my workflow
* choose a project on the left that I need to work on.
* select which branch to work on or click the new branch button
* hit the open in atom button

I change repositories quite often, an option to show a list on the left is needed instead of a drop-down that auto hides.

The horizontal branch view/commit view was a simple and nice overview of the project, this is what made me love it so much. A new branch button can be inside this.

Needs a light theme, everything blends in if I have something that is dark behind the app.

Currently, the layout and getting around is not quick and easy, and is missing features from the previous version.

Please add a link at the bottom of the website to download the old version, I cannot find it anywhere.

**Comment by j-f1** [comment]
> an option to show a list on the left is needed instead of a drop-down that auto hides

See #1593 for the issue tracking this.

**Comment by j-f1** [comment]
> Needs a light theme, everything blends in if I have something that is dark behind the app.

Interesting. This is the opposite of what we’ve been hearing theme-wise (most people want a darker theme ⚫️)

**Comment by shiftkey** [comment]
@massivelines thanks for the feedback. We can't make any guarantees as to how long it'll continue to work nor for how long the installer will be available but for now you can download it at https://github-windows.s3.amazonaws.com/GitHubSetup.exe.

As to your feedback:

> - I change repositories quite often, an option to show a list on the left is needed instead of a drop-down that auto hides.

As @j-f1 mentioned, this is in #1593 - we talked a bunch in there around the tradeoffs and why we changed this up in the Electron build. There's also a keyboard shortcut for <kbd>Ctrl + T</kbd> to quickly expand this repository list.

> - The horizontal branch view/commit view was a simple and nice overview of the project, this is what made me love it so much. A new branch button can be inside this.

The graph thing is not on our immediate roadmap (see #1634 for that discussion) but you can see the designs for our "Compare two branches" UI that we're about to start working on: https://github.com/desktop/desktop/issues/2639#issuecomment-333621771

Also, we're adding "Create new Branch" closer to the branches list in https://github.com/desktop/desktop/pull/3033 which is under review.

> - Needs a light theme, everything blends in if I have something that is dark behind the app.

This one is new, as I think most people assume the theme we have works as a "light" theme. Would you mind elaborating on this in a new issue, maybe with screenshots to contrast what you're seeing, so we can talk about this in more detail? Maybe there's some tweaks we can do.

**Comment by massivelines** [comment]
Thanks for the link, I got a new laptop and wanted to put it on it.

By 'light" theme, I really meant the dark bars at the top, maybe like a medium gray. Usually, dark themes are too dark for my setup, I prefer more grayish themes, like the JetBrains ui, even made a different atom ui, https://atom.io/themes/neutral-gray-ui

Check out how Adobe handles their colors and themes in their programs like Photoshop.
(edit > preferences > interface) It has four different brightnesses to choose from.
I know it will be way down the road but allowing us to create our own themes like atom would be pretty cool too.

I know in windows, the personalization settings menu lets you change the color of your start menu and other things. It also controls a 1 - 2px border around programs. I have noticed that some programs set this color and it cannot be changed by the personalization. It could help frame the program, giving it a contrast edge all around. On the ones that cannot be changed, if it is primarily a dark color at the top it is usually a lighter shade and for a light color at the top, it is a darker shade.

I am not sure how these will show up, it does depend on the brightness and colors of the monitor, so what I screenshot won't be what I see on my screen. I am running a color corrected monitor for design work which probably makes the top bar darker on me.

![dark](https://user-images.githubusercontent.com/10354555/31594972-0207a986-b1fe-11e7-93ce-8732d024bcf0.jpg)
![light](https://user-images.githubusercontent.com/10354555/31594973-0216eacc-b1fe-11e7-9a5a-62143097d1bc.jpg)
